packages/react-menu/src/components/MenuGroup/MenuGroup.types.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utils';
+
+/**
+ * {@docCategory MenuGroup}
+ */
+export type MenuGroupProps = ComponentProps & React.HTMLAttributes<HTMLElement> & {};
+
+/**
+ * {@docCategory MenuGroup}
+ */
+export interface MenuGroupState extends MenuGroupProps {
+  ref?: React.MutableRefObject<HTMLElement>;
```

Comment:
```suggestion
  ref: React.MutableRefObject<HTMLElement>;
```

I don't think that it's optional